packages/FormElement/stories/FormElement.stories.mdx

```diff
@@ -76,14 +71,14 @@ In addition, a `<label>` is rendered by the `<FormElement>` and is associated to
 ##### Result
 
 <FormElementStory>
+  <Tagline>Form Element with instructions component.</Tagline>
+  <br />
   <AccessibilityExample />
 </FormElementStory>
 
-### Alternate Layouts
-
-The FormElement can be used in a variety of formats. These can include FormElements with checkboxes, radios, listbox's etc.
+### Components
 
-#### Checkboxes
+The `<FormElement />` can be used with differant components to create specific input needs. Some of these can include components such as the checkbox or radio.
```

Comment:
```suggestion
The `<FormElement>` can be used with different components to fulfill a variety of input needs. This can include using basic input components such as the `<Select>`, `<Textarea>`, `<Checkbox>`,  etc.
```

Reply:
I've preferred `<FormElement />` because `<FormElement>` isn't valid JSX but not a hill to die on 😄